.github/workflows/build-test.yaml

```
@@ -1041,56 +1051,56 @@ jobs:
           EXIT_CODE=$?
           if [ $EXIT_CODE -ne 0 ]; then
             echo "------pods:"
-            kubectl -n $APP_SLUG get pods
+            kubectl -n "$APP_SLUG" get pods
             echo "------kotsadm logs"
-            kubectl logs -l app=kotsadm --tail=100 --namespace $APP_SLUG
+            kubectl logs -l app=kotsadm --tail=100 --namespace "$APP_SLUG"
             exit $EXIT_CODE
           fi
 
           COUNTER=1
-          while [ "$(./bin/kots get apps --namespace $APP_SLUG | awk 'NR>1{print $3}')" != "$APP_VERSION_LABEL" ]; do
-            COUNTER=$[$COUNTER +1]
+          while [ "$(./bin/kots get apps --namespace "$APP_SLUG" | awk 'NR>1{print $3}')" != "$APP_VERSION_LABEL" ]; do
+            ((COUNTER += 1))
             if [ $COUNTER -gt 120 ]; then
               echo "Timed out waiting for app to be installed with correct version label: $APP_VERSION_LABEL"
-              ./bin/kots get apps --namespace $APP_SLUG
+              ./bin/kots get apps --namespace "$APP_SLUG"
               echo "kotsadm logs:"
-              kubectl logs -l app=kotsadm --tail=100 --namespace $APP_SLUG
-              exit -1
+              kubectl logs -l app=kotsadm --tail=100 --namespace "$APP_SLUG"
+              exit 255
```

I think these can just be `1` instead of `-1`. Pretty sure we put `-1` by mistake.
```suggestion
              exit 1
```